### PR TITLE
[quest] ADDED: 'Strength of Soul' Priest Fake Rune Quest To QuestDB

### LINKED ISSUE #5491: [quest] 'Strength of Soul' Priest Fake Rune Quest Missing From QuestDB
## Description

The SoD Priest fake rune quest 'Strength of Soul' is not currently in the QuestDB. 

## Wowhead link

https://www.wowhead.com/classic/guide/season-of-discovery/classes/priest/strenght-of-soul-rune
https://www.wowhead.com/classic/spell=415739/strength-of-soul
https://www.wowhead.com/classic/item=211534/primal-insight

## Screenshots

N/a

## Questie & WoW version

v9.2.14
WoW Classic SoD - v1.15.052409

### PR DESCRIPTION
**Pull request by Sommos**
## Issue references

Fixes #5491 
Linked To #5443 

## Proposed changes

- ADDED: 'Strength of Soul' Priest Fake Rune Quest is now in the QuestDB, and should be accessible to users.

### CODE REVIEW

File: Database/Corrections/sodQuestFixes.lua [hunk 1]
```diff
@@ -2617,6 +2617,18 @@ function SeasonOfDiscovery:LoadQuests()
...
         },
+        [90161] = {
+            [questKeys.name] = "Strength of Soul",
+            [questKeys.startedBy] = {{3924,3750,3922,3748},{450003},{211534}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 23,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.PRIEST,
+            [questKeys.objectivesText] = {"Kill Thislefur for Primal Insight, then go up in to the tree, and use it whilst near the Thislefur Dreamcatcher, and receive the rune."},
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:2628]: You need to have two mediation buffs active.

**Review by BreakBB — CHANGES_REQUESTED** [review]

File: Database/Corrections/sodQuestFixes.lua [hunk 2]
```diff
@@ -2617,6 +2617,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -425216,
             [questKeys.zoneOrSort] = sortKeys.PRIEST,
         },
+        [90161] = {
+            [questKeys.name] = "Strength of Soul",
+            [questKeys.startedBy] = {{3924,3750,3922,3748},{450003},{211534}},
```
[the remark this answers is not in the document]
Comment (Sommos) [reply Database/Corrections/sodQuestFixes.lua]: Fixed in: https://github.com/Questie/Questie/pull/5492/commits/ae3d1990a7e603bb9b9178f2d4889753e7278597

File: Database/Corrections/sodQuestFixes.lua (hunk 1 above)
Comment (Sommos) [reply Database/Corrections/sodQuestFixes.lua]: Fixed in: https://github.com/Questie/Questie/pull/5492/commits/259891f40e236ff011396fe9821b18be3fc574f4

**Review by BreakBB — CHANGES_REQUESTED** [review]
This quest should also be hidden for now.

**Review by BreakBB — APPROVED** [review]

**Pull request merged**